application/values.yaml

```diff
@@ -92,13 +92,13 @@ deployment:
     # -- (string) Type of deployment strategy.
     # @section -- Deployment Parameters
     type: RollingUpdate
-    rollingUpdate:
+    # rollingUpdate:
       # -- (string) Max unavailable pods during update.
       # @section -- Deployment Parameters
-      maxUnavailable: 25%
+      # maxUnavailable: 25%
       # -- (string) Max surge pods during update.
       # @section -- Deployment Parameters
-      maxSurge: 25%
+      # maxSurge: 25%
```

Comment:
Can you please remove the documentation comments and pack the example values like in the initContainers sections below? Then, build the docs with `make build-docs` and then commit the resulting README. Thank you!